packages/shex-webapp/doc/shex-simple.html

```diff
@@ -312,8 +312,9 @@ <h2 class="status"> </h2>
             <div>
             <div id="shapeMap-tabs" class="droparea" style="width: 100%">
               <ul>
-                <li><a href="#textMap">Query</a></li>
-                <li><a href="#fixedMap-tab">Entities to check</a></li>
+                <li id="query"><a href="#textMap">Query Map</a></li>
+                <li id="queryEditor"><a href="#editMap-tab">Query Map Editor</a></li>
+                <li id="fixMap"><a href="#fixedMap-tab">Fixed Map</a></li>
```

Comment:
I think the IDs should match the tabs they link to – also, especially `query` is pretty generic. Perhaps `textMap-tab-header`/`editMap-tab-header`/`fixedMap-tab-header`?